response_operations_ui/views/collection_exercise.py

```diff
@@ -162,6 +163,16 @@ def _validate_collection_instrument():
                 "header": "Error: wrong file type for Collection instrument",
                 "message": "Please use XLSX file only"
             }
+        else:
+            # file name format is surveyId_period_formType
+            form_type = _get_form_type(file.filename) if file.filename.count('_') == 2 else None
+            if not form_type.isdigit() and len(form_type) != 4:
+                logger.debug('Invalid file format uploaded', filename=file.filename)
+                error = {
+                    "section": "ciFile",
+                    "header": "Error: invalid file name format for Collection instrument",
```

Comment:
🎨 Same here, can we title or lower case "Collection Instrument" (consistency would be lower)